Added GI, Profundity, Jabba. Split out fleet

dinkatoid opened the pull request.

benbaxter commented on src/data/events.ts:
```diff
@@ -79,28 +92,51 @@ export default new FarmCategory(
-            ],
-            [],
-        ),
+        // new Farm(
```
nit: The Farm is still available in git's history and in src/data/fleets.ts. There is no need to keep around commented code.

Thanks for putting together this PR!

dinkatoid commented:
I did this cause I had split out the fleet section into its own thing and I wasn't sure how you would feel about that. Is that OK, or do you want it mixed into events? I can remove the commented part and fix it up and send in an updated request.
________________________________
From: benbaxter ***@***.***>
Sent: Wednesday, September 14, 2022 7:32 AM
To: ThingEngineering/swgoh-status ***@***.***>
Cc: dinkatoid ***@***.***>; Author ***@***.***>
Subject: Re: [ThingEngineering/swgoh-status] Added GI, Profundity, Jabba. Split out fleet (PR #29)


@benbaxter commented on this pull request.

________________________________

In src/data/events.ts<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FThingEngineering%2Fswgoh-status%2Fpull%2F29%23discussion_r970746350&data=05%7C01%7C%7C10cfe44ac559494bead608da964d38cb%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637987555658313053%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=0ziEtjiiKhw6kxNkAfIi8177GiVwwKlqeX0uIpyZayo%3D&reserved=0>:

> -                new FarmUnit('TIE Fighter Pilot', 7, 13, 5),
-                new FarmUnit('Bossk', 7, 13, 5),
-                new FarmUnit('IG-88', 7, 13, 5),
-                new FarmUnit('Dengar', 7, 13, 5),
-                spacer,
-                new FarmUnit('Razor Crest', 7),
-                new FarmUnit('Slave I', 7),
-                new FarmUnit('IG-2000', 7),
-                new FarmUnit('Hound\'s Tooth', 7),
-                new FarmUnit('TIE Advanced x1', 7),
-                new FarmUnit('Imperial TIE Bomber', 7),
-                new FarmUnit('Imperial TIE Fighter', 7),
-            ],
-            [],
-        ),
+        // new Farm(


nit: The Farm is still available in git's history and in src/data/fleets.ts. There is no need to keep around commented code.

Thanks for putting together this PR!

—
Reply to this email directly, view it on GitHub<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FThingEngineering%2Fswgoh-status%2Fpull%2F29%23pullrequestreview-1107392582&data=05%7C01%7C%7C10cfe44ac559494bead608da964d38cb%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637987555658469834%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=3rsMy9IXwweScI73aBaX5%2BzmVMOm3H753i2hfEGIK20%3D&reserved=0>, or unsubscribe<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAEKOQ3YXVJOCWI6MJLARLN3V6HAWVANCNFSM6AAAAAAQJEBXDI&data=05%7C01%7C%7C10cfe44ac559494bead608da964d38cb%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637987555658469834%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=DAeOJvw%2BGrQm5iclUpglIu%2Bdk4PFjzbeUYBkz6u95tI%3D&reserved=0>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

benbaxter commented:
I do not have write permission to this repo, I was just leaving a
suggestion to clean up the code so the PR is in a final state to be
submitted.

(I was looking to add these events myself and saw you beat me to it! 😁)

On Tue, Oct 11, 2022 at 5:02 AM dinkatoid ***@***.***> wrote:

> I did this cause I had split out the fleet section into its own thing and
> I wasn't sure how you would feel about that. Is that OK, or do you want it
> mixed into events? I can remove the commented part and fix it up and send
> in an updated request.
> ________________________________
> From: benbaxter ***@***.***>
> Sent: Wednesday, September 14, 2022 7:32 AM
> To: ThingEngineering/swgoh-status ***@***.***>
> Cc: dinkatoid ***@***.***>; Author ***@***.***>
> Subject: Re: [ThingEngineering/swgoh-status] Added GI, Profundity, Jabba.
> Split out fleet (PR #29)
>
>
> @benbaxter commented on this pull request.
>
> ________________________________
>
> In src/data/events.ts<
> https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FThingEngineering%2Fswgoh-status%2Fpull%2F29%23discussion_r970746350&data=05%7C01%7C%7C10cfe44ac559494bead608da964d38cb%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637987555658313053%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=0ziEtjiiKhw6kxNkAfIi8177GiVwwKlqeX0uIpyZayo%3D&reserved=0
> >:
>
> > - new FarmUnit('TIE Fighter Pilot', 7, 13, 5),
> - new FarmUnit('Bossk', 7, 13, 5),
> - new FarmUnit('IG-88', 7, 13, 5),
> - new FarmUnit('Dengar', 7, 13, 5),
> - spacer,
> - new FarmUnit('Razor Crest', 7),
> - new FarmUnit('Slave I', 7),
> - new FarmUnit('IG-2000', 7),
> - new FarmUnit('Hound\'s Tooth', 7),
> - new FarmUnit('TIE Advanced x1', 7),
> - new FarmUnit('Imperial TIE Bomber', 7),
> - new FarmUnit('Imperial TIE Fighter', 7),
> - ],
> - [],
> - ),
> + // new Farm(
>
>
> nit: The Farm is still available in git's history and in
> src/data/fleets.ts. There is no need to keep around commented code.
>
> Thanks for putting together this PR!
>
> —
> Reply to this email directly, view it on GitHub<
> https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FThingEngineering%2Fswgoh-status%2Fpull%2F29%23pullrequestreview-1107392582&data=05%7C01%7C%7C10cfe44ac559494bead608da964d38cb%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637987555658469834%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=3rsMy9IXwweScI73aBaX5%2BzmVMOm3H753i2hfEGIK20%3D&reserved=0>,
> or unsubscribe<
> https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAEKOQ3YXVJOCWI6MJLARLN3V6HAWVANCNFSM6AAAAAAQJEBXDI&data=05%7C01%7C%7C10cfe44ac559494bead608da964d38cb%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637987555658469834%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=DAeOJvw%2BGrQm5iclUpglIu%2Bdk4PFjzbeUYBkz6u95tI%3D&reserved=0
> >.
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ThingEngineering/swgoh-status/pull/29#issuecomment-1274353468>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABHFTLRV6W5KUZGZACLIUJLWCUUIRANCNFSM6AAAAAAQJEBXDI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Pelemane reviewed (dismissed):
looks good
